vizro-core/src/vizro/_vizro.py

```diff
@@ -108,9 +108,10 @@ def build(self, dashboard: Dashboard):
 
         # Note that model instantiation and pre_build are independent of Dash.
         self._pre_build()
-
         self.dash.layout = dashboard.build()
 
+        # Add data-bs-theme attribute for 404 error page
```

Comment:
```suggestion
        # Add data-bs-theme attribute that is always present, even for pages without theme selector, i.e. the Dash "Loading..." screen.
```

Please can we put this new bit of code directly after these lines instead? I think it fits better in there.
```
if dashboard.title:
            self.dash.title = dashboard.title
```